developer-docs-site/docs/concepts/wallet-adapter-concept.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Aptos Wallet Adapter"
+id: "wallet-adapter-concept"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Wallet Adapter
+
+Aptos provides an [Aptos Wallet Adapter](https://github.com/aptos-labs/aptos-wallet-adapter) monorepo for wallet and dapps creators to ease development and ensure a smooth process in building projects on the Aptos network.
```

Comment:
these paragraphs don't really describe the problem that the wallet adapter is addressing...

1. There are a lot of wallets in the ecosystem
2. Supporting each one is a burden but not supporting them is risky for projects
3. Integration with wallets can be a complex process that requires a lot of testing

A wallet adapter solves these pain points....

or something

Reply:
we explain it here in details, https://github.com/aptos-labs/aptos-core/pull/5812/files#diff-28a8dc8c00ae95b5f3d12241e42fbdcaf8e022df85c141d748d8de3cb2113001R8-R23

but we can also add some of that on that page! maybe benefits for both wallets and dapps